### PR TITLE
fix for #5487

### PR DESCRIPTION
**Pull request by enthus1ast**

### CODE REVIEW

File: lib/pure/collections/tables.nim [hunk 1]
```diff
@@ -231,7 +231,7 @@ iterator allValues*[A, B](t: Table[A, B]; key: A): B =
     h = nextTry(h, high(t.data))
 
 proc hasKey*[A, B](t: Table[A, B], key: A): bool =
-  ## returns true iff `key` is in the table `t`.
+  ## returns true if `key` is in the table `t`.
```
Comment (Araq) [inline lib/pure/collections/tables.nim]: 'iff' is not a typo! it means "if and only if".

Comment (enthus1ast) [inline lib/pure/collections/tables.nim]: really Araq?

Comment (enthus1ast) [inline lib/pure/collections/tables.nim]: will you only accept it when i change it back?

Comment (Araq) [inline lib/pure/collections/tables.nim]: Really. https://en.wikipedia.org/wiki/If_and_only_if

**Pull request merged**